jina/importer.py

```diff
@@ -95,6 +96,57 @@ def __exit__(self, exc_type, exc_val, traceback):
                 return True  # suppress the error
 
 
+class ModuleLoader(Loader):
+    """A custom module loader"""
+
+    @classmethod
+    def exec_module(cls, module):
+        """load module into memory
+
+        :param module: a namespace module
+        """
+        load_path = module.__spec__.origin
+        init_file = '__init__.py'
+        if load_path.endswith(init_file):
+            module.__path__ = [load_path[: -len(init_file) - 1]]
+
+
+_loader = ModuleLoader()
+
+
+class PathFinder:
```

Comment:
may be better to rename this to JinaExecPathFinder, according to https://pyoxidizer.readthedocs.io/en/stable/oxidized_importer_meta_path_finders.html there is a PathFinder class involved and could become confusing?

Reply:
thanks for your suggestions. I have renamed it as `ExtensionPathFinder`